guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -147,13 +147,6 @@ endif::[]
 ----
 # {foreman-maintain} service restart
 ----
-. Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
```

Comment:
@jlsherrill  - this guide should be still used for upgrades to Foreman 2.5 - does this still apply to the Foreman scenario?

Reply:
Steps 8 and 9 in the foreman documentation are identical to Steps 10 and 11, should the steps be repeated, or is it an error?

Reply:
I see now. They were added in error here: https://github.com/theforeman/foreman-documentation/pull/490/files#diff-c24ec779d7504e0ce3bb6f2834002cdb6abff1023e249d73704797c54ac4a7d2R134 
Resolving this, thanks!